tests/test_cli.py

```diff
@@ -207,11 +207,11 @@ def get_expected_locations_results(include_parent):
             ],
             "rows": [
                 ["id1", "2020-04-01 21:57:26", "d1", "eid1",
-                 "2020-04-01 21:58:23", "11.2", "ld1", "lid1", "lt1",
+                 "2020-04-01 21:58:23", "11.2", '{"p1": "ld1", "id": "id1.location_data"}', "lid1", "lt1",
```

Comment:
this is an example of the id property that gets injected

Reply:
It was a surprise to me, but the 'id' comes from this line:
https://github.com/dimagi/commcare-export/blob/master/commcare_export/env.py#L203

It seems to be recording the path to the dictionary within its enclosing object. I would leave it as is for now because I don't know how to avoid it for location_data without affecting its intended use.

Reply:
thanks for chasing this down!